conf/janus.transport.websockets.jcfg.sample

```diff
@@ -38,8 +38,11 @@ admin: {
 }
 
 # Certificate and key to use for any secure WebSocket server, if enabled (and passphrase if needed).
+# You can also disable insecure protocols and ciphers by configuring the
+# 'ciphers' property accordingly (no limitation by default).
 certificates: {
 	#cert_pem = "/path/to/cert.pem"
 	#cert_key = "/path/to/key.pem"
 	#cert_pwd = "secretpassphrase"
+	#ciphers = "DEFAULT"
```

Comment:
I'd probably set a reasonable default here, like the one we have in the HTTP config file; something that gives people an idea of what the syntax looks like. The example you made in the description may be too long for that, but something along those lines maybe.

Reply:
I've updated the example cipher string to the two ECDHE ciphers that are supported universally across all modern browsers, and I've linked out to the OWASP page with additional details on cipher strings and browser compatibility.